### DISCUSSION TITLE
test QA

### DISCUSSION THREAD

**Discussion by lyrieek** [ZhanChaoHan/Apache#16] [Q&A] [answered]
QA

**Comment by ZhanChaoHan** [comment, marked as answer]
@Select(value = "SELECT computerId,computerName,computerPrice,madeTime FROM computer\r\n"
			+ "	     WHERE (computerPrice >= #{price}) ORDER BY computerPrice ASC")
	List<Computer> selectPageByAnn(@Param("price") Long price);
